exercises/card_counting.livemd

```diff
@@ -176,7 +181,20 @@ You could omit the **normal** cards or allow **high** and **low** cards to balan
 Enter your solution below.
 
 ```elixir
+random_card1 = Enum.random(1..13)
```

Comment:
Did you come up with this solution from scratch? If so, well done! It looks identical to the example.